CONTRIBUTING.md

```diff
@@ -106,7 +106,19 @@ $ pip install -r requirements/development.txt
 If you are running in GitHub Codespaces or other Ubuntu/Debian environment, you may need to install additional libraries. Do this by running:
 
 ```bash
-make install-codespaces
+$ make install-codespaces
+```
+
+### Building the Dashboard
```

Comment:
Building from the project root *should* do this already, but it seems to be broken finding `run-s`.  Looking closer at it, I think we're missing a `yarn install` in the root's `build` target.  Try adding that instead of this separate step in the docs?  The less manual steps, the better.

Reply:
Can I add a script to just build the entire project?

Reply:
As part of `make` run from the root?  That's reasonable - the `docker` target does all of it, including the ai project stuff.  I think the only part that's missing for a complete setup in the `build` target is the venv install.

Reply:
@Mcilie and I discussed this offline - he's going to make the changes to make the contributing guide work correctly and move on to #247.  Can discuss doing a more robust "one shot" build as part of a future issue.

Reply:
@Mcilie minor tweak, for code/shell blocks in markdown, it's considered better to set the 
```<type>, E.g. ```bash
and then exclude the prompt character "$".
This way people can easily copy and paste exactly what going into the prompt.